src/base/fileconfig.lua

```diff
@@ -152,7 +152,11 @@
 --
 
 	function fileconfig.getconfig(fcfg, cfg)
-		return fcfg.configs[cfg]
+		if fcfg.configs ~= nil then
```

Comment:
@starkos this is the second time I've seen this change be suggested, what is the preferred way to fix the issue of this function indexing `nil`?

Reply:
@arthurdead Can you provide a minimal example of the setup that caused you to need this fix? I'm not sure how to reproduce the problem.